### PR TITLE
Added a new project idea (Web development) in the project-ideas.json

### LINKED ISSUE #2088: [ADD] <Web Development Project>
### Category Name

Resources

### Subcategory Name

Projest-Ideas

### Additional Context

The link I provided contains guided Web-Development projects which are free.

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I have checked that this category does not exist
- [X] I have checked that this subcategory does not exist
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by the-shivam-gupta**
## Fixes Issue
Closes #2088

### CODE REVIEW

**Comment by Anmol-Baranwal** [comment]
@the-shivam-gupta 
You should add a profile photo because these are default ones, and it gives off a vibe like the contributor is new to GitHub. So, adding a photo would be good since your profile is decent enough. (GitHub profile)

**Comment by the-shivam-gupta** [comment]
> @the-shivam-gupta 
> You should add a profile photo because these are default ones, and it gives off a vibe like the contributor is new to GitHub. So, adding a photo would be good since your profile is decent enough. (GitHub profile)

Thanks for the compliment! I'll update my profile soon. Honestly, I don't have a great photo of myself, which is why it's missing 😅.

**Comment by Anmol-Baranwal** [comment]
> > @the-shivam-gupta
> > You should add a profile photo because these are default ones, and it gives off a vibe like the contributor is new to GitHub. So, adding a photo would be good since your profile is decent enough. (GitHub profile)
> 
> Thanks for the compliment! I'll update my profile soon. Honestly, I don't have a great photo of myself, which is why it's missing 😅.

You don't need to add your own photo. Anything that you find cool 😅

You can check some avatars [here](https://github.com/Anmol-Baranwal/Cool-GIFs-For-GitHub/blob/main/README.md#--cool-avatars-for-github-profile--)

File: database/resources/project-ideas.json [hunk 1]
```diff
@@ -103,5 +103,12 @@
     "url": "https://www.interviewbit.com/blog/cyber-security-projects/",
     "category": "resources",
     "subcategory": "project-ideas"
+  },
+  {
+    "name": "Web Dev Projects",
+    "description": "These projects will help you grasp the skills of a full-stack developer, and you can also add these projects to your resume. ",
```
Comment (CBID2) [inline database/resources/project-ideas.json:109]:
```suggestion
    "description": "These projects will help you learn the skills to become a full-stack developer. You can even reference them in your resume. ",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @the-shivam-gupta! :) Your PR is off to a great start! :) I pointed out some proofreading errors in your description that could be improved.

**Review by the-shivam-gupta — COMMENTED** [review]
@CBID2  done

**Review by CBID2 — APPROVED** [review]
Looks good to me @the-shivam-gupta! :) @Anmol-Baranwal, @aftabrehan, it's your turn.

**Review by Anmol-Baranwal — APPROVED** [review]
> There are few great projects.

🚀 Looks good to me.

**Review by aftabrehan — APPROVED** [review]
Helpful resource, good job @the-shivam-gupta!

**Pull request merged**